scenariolib/visit.go

```diff
@@ -114,7 +115,14 @@ func NewVisit(_searchtoken string, _uatoken string, _useragent string, language
 }
 
 func buildUserEmail(c *Config) string {
-	return fmt.Sprint(c.FirstNames[rand.Intn(len(c.FirstNames))], ".", c.LastNames[rand.Intn(len(c.LastNames))], c.Emails[rand.Intn(len(c.Emails))])
+	firstName := c.FirstNames[rand.Intn(len(c.FirstNames))]
+	lastName := c.LastNames[rand.Intn(len(c.LastNames))]
+	emailSuffix := c.Emails[rand.Intn(len(c.Emails))]
+	if lastName == "" {
```

Comment:
I actually had a demand from spaquet to name the bot uabot@coveo.com.  Could have called it ua.bot@coveo.com.  But in fact I can revert those changes because as an anonymous user, the username is not persisted.